tikv-client/src/main/java/com/pingcap/tikv/util/ChannelFactory.java

```diff
@@ -84,8 +83,7 @@ private ManagedChannel addrToChannel(String addressStr) {
       try {
         sslContext = sslContextBuilder.build();
       } catch (SSLException e) {
-        LOG.error("build sslContextBuilder false", e);
-        return null;
+        throw new TiKVException("build sslContextBuilder false", e);
```

Comment:
Failed to build sslContextBuilder

Reply:
Thanks for your advice. @xuanyu66 . I have changed it.